lib/private/Security/Signature/SignatureManager.php

```diff
@@ -353,6 +353,7 @@ private function insertSignatory(Signatory $signatory): void {
 		$time = time();
 		$signatory->setCreation($time);
 		$signatory->setLastUpdated($time);
+		$signatory->setMetadata($signatory->getMetadata()); // trigger insert on field metadata using current or default value
```

Comment:
Since this is the default value on the object, it will not mark the field as changed and therefore not write it to the database.
That being said, the default value on the object should/must be the one in the database, so `null` here. And then there you would do:
```suggestion
		$signatory->setMetadata([]);
```

Reply:
$metadata is now `null` by default